lxterminal.yaml

```diff
@@ -0,0 +1,11 @@
+name: lxterminal
+config-opts:
+  - --enable-man 
```

Comment:
Man pages are not exported.

Reply:
> Man pages are not exported.

Man pages are accessible via terminal. See my comment below.